README.md

```diff
@@ -120,6 +120,11 @@ Flame is a minimalist 2D game engine for Flutter that provides a nice set of som
 
 - KINGA - [Android](https://play.google.com/store/apps/details?id=com.knoeyes.games.kinga), [iOS](https://apps.apple.com/us/app/kinga/id1506422810) - Protect the cookie from the pesky flies. By [Kevin Omyonga](https://kevinomyonga.com)
 
+#### Dino Run
```

Comment:
I don't think Dino run is a category, add your game under some existing category (eg Casual).